utils/get-current-version.sh

```diff
@@ -1,7 +1,4 @@
 #!/bin/bash
 
-ROOT_DIR=$(git rev-parse --show-toplevel)
-CSV="$ROOT_DIR/bundle/manifests/compliance-operator.clusterserviceversion.yaml"
-
-OLD_VERSION=$(yq '.spec.version' "$CSV")
+OLD_VERSION=$(git describe --tags --abbrev=0 | tr -d v)
```

Comment:
Based on [#499 ](https://github.com/ComplianceAsCode/compliance-operator/pull/499#discussion_r1562915918) we can keep the letter`v`.
And `compliance-operator.` needs to be added to the `sed` command in the Makefile.